content/applications/services/fsm/default_warehouse.rst

```diff
@@ -0,0 +1,40 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
+
+With this feature configured, the products in sales orders created during field interventions are
+always pulled from the same warehouse, keeping the inventory accurate.
+
+.. seealso::
+   :doc:`../../inventory_and_mrp/inventory`
+
+Prerequisites
+=============
+
+To set up a user default warehouse, configure the following settings first:
+
+ #. Open the **Inventory** app, then go to :menuselection:`Configuration --> Settings`. Activate
+    **Storage Locations** and **Multi-Step Routes**.
+ #. Have more than one warehouse in your database.
```

Comment:
A link to how to do it or to check out the inventory documentation can be helpful.
Or else maybe don't mention it at all if we don't know how to do it.

Reply:
In the end I just explained how to do it & added a link to inventory docs , does that make sense?